src/test/java/ludogorie_soft/reservations_platform_api/controller/PropertyControllerTest.java

```diff
@@ -0,0 +1,158 @@
+package ludogorie_soft.reservations_platform_api.controller;
+
+import ludogorie_soft.reservations_platform_api.dto.PropertyRequestDto;
+import ludogorie_soft.reservations_platform_api.dto.PropertyResponseDto;
+import ludogorie_soft.reservations_platform_api.service.PropertyService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class PropertyControllerTest {
```

Comment:
It is better to have unit tests with Mocks for Service classes and integration tests with MockMvc for controllers - https://howtodoinjava.com/spring-boot2/testing/spring-boot-mockmvc-example/

Reply:
Wildcard imports have been removed.

Reply:
Added Integration test for controller